packages/vue/README.md

```diff
@@ -0,0 +1,131 @@
+# @synergy-design-system/vue
+
+This package provides [vue.js](https://vuejs.org/) wrappers for [Synergy Web Components](https://github.com/SickDesignSystem/synergy/tree/main/packages/components).
+
+This package aims to provide an improved developer experience in vue applications:
+
+- Provides two way data binding and `v-model`
+- Autocompletion and Types
+- Better custom event handling of `synergy` events
+
+## Getting started
+
+### 1. Package installation
+
+Run the following steps to install the required packages.
+
+```bash
+# Install the base library and required css files
+npm install --save @synergy-design-system/vue @synergy-design-system/tokens
+```
+
+> ⚠️ Note we do **not** ship vue in this package.
+> You will have to install it by yourself first!
+
+### 2. Add the wanted theme to your application
+
+The components will not display correctly without the needed theme. Please include either light or dark theme in your application, for example in a newly installed vue application:
+
+```ts
+// src/main.ts
+// Add this line to enable the light theme for your application
+import '@synergy-design-system/tokens/dist/themes/light.css';
+
+import { createApp } from 'vue'
+import App from './App.vue'
+
+createApp(App).mount('#app')
+```
+
+### 3. Importing and rendering components
+
+You may now use the components by importing them from the `@synergy-design-system/vue` package and rendering them in your own vue components.
+
+```html
+<script lang="ts">
```

Comment:
I would add "setup" here in this line. Tried to copy paste this in a project, but it didn`t work, that the button is shown correctly till I added the setup.
Before adding setup: 
![image](https://github.com/SickDesignSystem/synergy/assets/115139104/5716ce78-121d-42a1-a65c-021e3f815269)

After adding setup:
![image](https://github.com/SickDesignSystem/synergy/assets/115139104/b3216ac2-8ea5-482a-bb20-60d0298c6e69)

```suggestion
<script setup lang="ts">
```